### ISSUE TITLE
tf.keras.losses has no attribute 'SparseCategoricalCrossentropy'

### ISSUE THREAD

**Issue by zhjw0927** [tensorflow/tensorflow#42353]
`model.compile(optimizer='adam',
            loss=tf.keras.losses.SparseCategoricalCrossentropy(from_logits=True),
            metrics=['accuracy'])`
Use above code, something went wrong:
![image](https://user-images.githubusercontent.com/42128459/90222246-48dcf400-de3e-11ea-93bc-2a5c8ecd8ecc.png)
Looking forward for your help.
Thank you!

**Comment by zhjw0927** [comment]
My tf version is 1.3.

**Comment by Saduf2019** [comment]
@zhjw0927 
Is there any particular reason for using this old version of tensorflow, support is available from tf 1.5 and 2.x, please upgrade to later versions and let us know if you still face the error in 2.x
Please refer to below links:
[link](https://www.tensorflow.org/api_docs/python/tf/keras/losses/SparseCategoricalCrossentropy)
[link1](https://www.tensorflow.org/versions/r2.0/api_docs/python/tf/keras/losses/SparseCategoricalCrossentropy) #26012 

In case the error still persist in 2.x version as well, please provide with stand alone indented code or a colab gist with the error for us to replicate.

**Comment by Aryan05** [comment]
Try to install the latest version of TensorFlow, maybe that might fix your error

**Comment by jvishnuvardhan** [comment]
@zhjw0927 This is already fixed in recent `tf-nightly`. This will be available in stable TF2.4 version. 
I am closing this issue as it was resolved. Feel free to reopen if this issue persists in `TF2.4` when released. Thanks!

**Comment by jvishnuvardhan** [comment]
This is resolved in recent `tf-nightly`. Please feel free to reopen if you notice the issue. 
This is available in stable `TF2.4` in the near future. Thanks!

**Comment by mawenju203** [comment]
Please replace 'loss=tf.keras.losses.SparseCategoricalCrossentropy(from_logits=True)' with loss=loss='sparse_categorical_crossentropy' as mentioned in https://github.com/tensorflow/tensorflow/issues/42045#issuecomment-674232499.
